modal/tests/runner/endpoints/test_completion.py

```diff
@@ -29,3 +29,12 @@ async def test_completion_streaming():
 
     output = await completion(prompt, stream=True)
     assert output is not None
+
+
+@pytest.mark.integration
+@pytest.mark.asyncio
+async def test_large_prompt():
+    """Should accept & complete a large prompt"""
+    prompt = get_words_from_file(n=2500)
+    output = await completion(prompt, max_tokens=5000)
+    assert output is not None
```

Comment:
dont love that these are basically only asserting a request was successful, but fine for now i suppose.  might be a bit awk to assert against what the model is generating

Reply:
Ooh interesting - how else and what other aspect of an LLM endpoint would you test from an e2e?

Reply:
honestly not too sure :) just a gut reaction

perhaps just sanity checking that something of substance was actually generated -- not just parroting back the prompt, a response of at least some length/complexity, etc

def gets a little fuzzy/soft, but thats kinda where my initial thinking is at (potentially naive)